contracts/LoadTester.sol

```diff
@@ -561,18 +561,23 @@ contract LoadTester {
         }
         return result;
     }
+
     function testDIFFICULTY(uint x) public returns(uint) {
         inc();
         uint result = 0xDEADBEEF0044;
         assembly {
             let v := 0
             for { let i := 0 } lt(i, x) { i := add(i, 1) }
             {
-                v := difficulty()
+                // NOTE: Post Paris, the difficulty() instruction has been supplanted for prevrandao() 
+                // Source: https://eips.ethereum.org/EIPS/eip-4399
+                // v := difficulty()
+                v := prevrandao()
```

Comment:
Very interesting... This is a weird refactor tbh. I.e. it's still named `DIFFICULTY` but it looks like it's still `0x44` opcode. I think this if okay for now but we should talk about changing the name.